wordpress-mobile/WordPress-Android
# Issue/13326 fix threats status progress labels

Parent #13326

This PR 
- Adds different progress labels (for showing "Fixing threat" and threat info) to the `jetpack_list_progress_item` layout and updates corresponding `ProgressState`.
- Hides action buttons when threats are fixing.

Before | After
-------|------
<img width="160" alt="before" src="https://user-images.githubusercontent.com/1405144/105670433-1106cc00-5f07-11eb-9dd2-4ad0d5f644d3.png"> | <img width="169" alt="after" src="https://user-images.githubusercontent.com/1405144/105670490-21b74200-5f07-11eb-99c8-2820593fe864.png">

To test:

Prerequisite:

- Make sure both Scan feature flag and MySiteImprovements flag are set to on in the App settings.
- You have access to WP.com account with a site having scan capability and threats (e.g. https://pressable-jetpack-daily-scan.mystagingwebsite.com). Server credentials should be set on the site.
- A threat is added to the site. In case you run out of threats, re-install bad calendar plugin from here and re-run scan.

1. Login to the app using above WP.com account and select the site on the My Site tab
2. Click scan menu item on My Site
3. Click fix threats button on the scan screen + ok in the confirmation dialog
4. Notice that

   - Fix threats state and info labels are shown  in the progress layout
   - Action buttons are not shown

PR submission checklist:
